amis/packer_ubuntu1804.json

```diff
@@ -167,7 +167,13 @@
     {
       "type": "shell",
       "inline": [
-        "sudo flock $(apt-config shell StateDir Dir::State/d | sed -r \"s/.*'(.*)\\/?'$/\\1/\")/daily_lock systemctl stop apt-daily.timer apt-daily.service apt-daily-upgrade.timer apt-daily-upgrade.service"
+        "sudo flock $(apt-config shell StateDir Dir::State/d | sed -r \"s/.*'(.*)\\/?'$/\\1/\")/daily_lock systemctl disable --now apt-daily.timer apt-daily.service apt-daily-upgrade.timer apt-daily-upgrade.service",
+      ]
+    },
+    {
+      "type": "shell",
```

Comment:
why not doing this in the same previous shell?

Reply:
moved